recipes/mdnsresponder/all/conandata.yml

```diff
@@ -5,23 +5,23 @@ sources:
   "1310.140.1":
     url: "https://github.com/apple-oss-distributions/mDNSResponder/archive/mDNSResponder-1310.140.1.tar.gz"
     sha256: "ecb0043ffc5a3cbf4f43da0298e351d787654d980b291d1cb567c6ddee9dc983"
+  "1790.80.10":
+    url: "https://github.com/apple-oss-distributions/mDNSResponder/archive/mDNSResponder-1790.80.10.tar.gz"
+    sha256: "ffb990231639dfe8c7f0b15d0665206a9013f1aacfb2cbb934eb87f6a57f38b7"
 patches:
   "878.200.35":
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/DLLStub.c.patch"
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/mDNSResponder.sln.patch"
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/mDNSWin32.c.patch"
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/Service.c.patch"
+    - patch_file: "patches/878.200.35/DLLStub.c.patch"
+    - patch_file: "patches/878.200.35/mDNSResponder.sln.patch"
+    - patch_file: "patches/878.200.35/mDNSWin32.c.patch"
+    - patch_file: "patches/878.200.35/Service.c.patch"
+    - patch_file: "patches/878.200.35/upgrade-windows-toolset.patch"
   "878.200.35-opt":
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/opt/permit-over-long-service-types.patch"
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/opt/poll-rather-than-select.patch"
-    - base_path: "source_subfolder"
-      patch_file: "patches/878.200.35/opt/unicast.patch"
+    - patch_file: "patches/878.200.35/opt/permit-over-long-service-types.patch"
+    - patch_file: "patches/878.200.35/opt/poll-rather-than-select.patch"
+    - patch_file: "patches/878.200.35/opt/unicast.patch"
   "1310.140.1":
-    - base_path: "source_subfolder"
-      patch_file: "patches/1310.140.1/dns_sd.c.patch"
+    - patch_file: "patches/1310.140.1/dns_sd.c.patch"
+  "1790.80.10":
+    - patch_file: "patches/1790.80.10/Resolve-compile-problem-on-linux-platform.patch"
```

Comment:
Could you please update conandata.yml by adding patch fields like `patch_description` and `patch_type` https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patches-fields

Patching source files is not allowed, but in this specific case is related to compiler definition. It would be great describing it to make it clear to future reviews.

Reply:
Thanks for your guidance. I have added `patch_description` and `patch_type`.